website/docs/r/automation_software_update_configuration.html.markdown

```diff
@@ -0,0 +1,211 @@
+---
+subcategory: "Automation"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_software_update_configuration"
+description: |-
+  Manages a Automation.
+---
+
+# azurerm_automation_software_update_configuration
+
+Manages a Automation.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "example-rg"
+  location = "East US"
+}
+
+resource "azurerm_automation_account" "test" {
+  name                = "example"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku_name            = "Basic"
+}
+resource "azurerm_automation_software_update_configuration" "example" {
+  name                    = "example"
+  resource_group_name     = azurerm_resource_group.test.name
+  automation_account_name = azurerm_automation_account.test.name
+  operating_system        = "Linux"
+  linux {
+    classification    = "Security"
+    excluded_packages = ["apt"]
+    included_packages = ["vim"]
+    reboot_setting    = "IfRequired"
+  }
+  duration = "PT2H2M2S"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Automation. Changing this forces a new Automation to be created.
+
+* `automation_account_name` - (Required) The name of the automation account in which the Schedule is created. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Automation should exist. Changing this forces a new Automation to be created.
```

Comment:
these can become `automation_account_id`?

Reply:
yes, they can. but all other existing resources under Automation are using these two properties to specify the automation account. so I keep this resource the same as others.

Reply:
They should change to use ID, so can we place use ID here and maybe add comments to the other ones that need to be updated